# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

geometry_msgs/Pose2D pose
geometry_msgs/Pose2D poseVariance
float32 width
float32 widthVariance
